src/redux/features/plantings/plantingsSelectors.ts

```diff
@@ -63,29 +69,60 @@ export const selectPlantingProgress = createSelector(
                 }))
             ),
             totalPlants: totalPlantsBySite[ps.id],
+            ...(plantingSiteProject
+              ? {
+                  projectName: plantingSiteProject.project_name,
+                  projectId: plantingSiteProject.project_id,
+                }
+              : {}),
           };
         });
     }
   }
 );
 
 // selector to search plantings
+export type PlantingProgress = {
+  plantingCompleted: boolean;
+  plantingSize: string;
+  projectName: string;
+  siteId: number;
+  siteName: string;
+  subzoneId: number;
+  subzoneName: string;
+  targetPlantingDensity: number;
+  totalPlants: number;
+  totalSeedlingsSent: number;
+  zoneId: number;
+  zoneName: string;
+};
+
 export const searchPlantingProgress = createSelector(
   [
-    (state: RootState, query: string, plantingCompleted?: boolean, siteName?: string) => selectPlantingProgress(state),
-    (state: RootState, query: string, plantingCompleted?: boolean, siteName?: string) => query,
-    (state: RootState, query: string, plantingCompleted?: boolean, siteName?: string) => plantingCompleted,
-    (state: RootState, query: string, plantingCompleted?: boolean, siteName?: string) => siteName,
+    (state: RootState, query: string, filters: Record<string, SearchNodePayload>) => selectPlantingProgress(state),
+    (state: RootState, query: string, filters: Record<string, SearchNodePayload>) => query,
+    (state: RootState, query: string, filters: Record<string, SearchNodePayload>) => filters,
   ],
-  (plantingProgress, query, plantingCompleted, siteNameSelected) => {
+  (plantingProgress, query, filters) => {
```

Comment:
guessing the client passes in a filters object managed in a state variable or memoized const

Reply:
Exactly